.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+  - "10"
+before_install:
+  - npm install -g eslint
```

Comment:
- is it really necessary to make this in the `before_install` stage instead of the `install`?
- eslint is already part of the `package.json`, if you install it globally, it might be a different version then in the package.json which can make different result

Reply:
fixed